budget_control/models/budget_period.py

```diff
@@ -532,9 +535,11 @@ def _set_budget_info_amount(self, source, domain, kwargs):
             kpi_matrix, kpi_lines, period
         )
         budget_info[source] = amount
-        # Commit
-        budget_info["amount_commit"] = 0.0
-        return budget_info
```

Comment:
Note that, in the original code, I do not return `budget_info`, because I intend to use `_set_budget_info_amount` to set value into budget_info dict directly.

This works because python always pass variable by reference (not by value).